index.html

```diff
@@ -1,117 +1,283 @@
-<!DOCTYPE html>
-<html lang="en">
+﻿<!DOCTYPE html>
+<html lang = "en">
 <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />
+    <meta name="description" content="" />
+    <meta name="author" content="" />
+    <!--[if IE]>
+        <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+        <![endif]-->
+    <title>ChariTech - Hackers at Cambridge</title>
+    <link rel="icon" href="assets/img/icon.png">
+    <!-- BOOTSTRAP CORE STYLE CSS -->
+    <link href="assets/css/bootstrap.css" rel="stylesheet" />
+    <!-- Font-Awesome Icons Styles -->
+    <link href="assets/css/font-awesome.css" rel="stylesheet" />
+     <!-- Pretty Photo For PopUp Styles -->
+    <link href="assets/css/prettyPhoto.css" rel="stylesheet" />
+    <!-- CUSTOM STYLE CSS -->
+    <link href="assets/css/custom.css" rel="stylesheet" />    
+    <!-- GOOGLE FONT -->
+    <link href='http://fonts.googleapis.com/css?family=Open+Sans' rel='stylesheet' type='text/css' />
 
-    <link rel="manifest" href="/manifest.json" />
-    <meta name="theme-color" content="#ffffff" />
-
-    <!--Import Google Icon Font-->
-    <link href="http://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
-    <!--Import materialize.css-->
-    <link type="text/css" rel="stylesheet" href="materialize/css/materialize.min.css"  media="screen,projection"/>
-
-    <!--Let browser know website is optimized for mobile-->
-    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
-
-    <link rel="stylesheet" type="text/css" href="style.css">
-  <title>Charitech</title>
 </head>
-<body>
-    <div class="container">
-        <div class="row">
-            <div class="col s6 m4 center-align">
-                <div class="card-content">
-                    <iframe src="http://free.timeanddate.com/countdown/i5mknyo5/n1234/cf11/cm0/cu4/ct0/cs1/ca0/co0/cr0/ss0/cac000/cpc000/pct/tcfff/fn2/fs100/szw448/szh189/iso2017-03-17T10:00:00" allowTransparency="true" frameborder="0" width="304" height="26"></iframe>
-                </div>
+<body  >
+    <div id="nav-wrapper" >
+           <div class="navbar navbar-inverse " >
```

Comment:
This is just a nit, but there are quite a few whitespace inconsistencies in the html and css. As an example, this line uses different indentation to the surrounding lines has whitespace after the last CSS class, and has whitespace before the end of the tag.

Don't waste time combing through it line by line, though. There are "beautifier" tools that you can run your code through to tidy it up automatically. There are even some that you can run automatically right in your editor as extensions.